### DISCUSSION TITLE
NegativeDimensionError: one border dimension is larger than half of the respective page dimension

### DISCUSSION THREAD

**Discussion by DominikBritz** [ocrmypdf/OCRmyPDF#1292] [Q&A]
I'm getting the error message below when I want to upload a specific PDF to paperless-ngx via the web UI. I'm on paperless-ngx 2.7.2.

`NegativeDimensionError: one border dimension is larger than half of the respective page dimension`

Routed here by https://github.com/paperless-ngx/paperless-ngx/discussions/6346

Please find the document attached.

[2010-08-22 Braun Series 5 550 Rasierer.pdf](https://github.com/ocrmypdf/OCRmyPDF/files/14930545/2010-08-22.Braun.Series.5.550.Rasierer.pdf)

**Comment by ToXinE** [comment]
same with lots of docs

**Comment by Andreas-737** [comment]
I have the same problem. Sometimes it works, sometimes it doesnt (same scanner, same configuration). If i open the scanned pdf and print it to pdf, upload is working fine...

**Comment by Jonas-Sander** [comment]
Did anyone find any solution? This often happens to me with scanned receipts (very long, but not very wide)

**Comment by ovizii** [comment]
Just hit the same problem and found this thread without any hints on how to fix.
I can confirm it happened with a very long and narrow scanned original (receipt-like format). 
What do we do? Add white background and make it less "slim"?
